doc/api/doxy-api.conf.in

```diff
@@ -62,7 +62,6 @@ INPUT                   = @TOPDIR@/doc/api/doxy-api-index.md \
                           @TOPDIR@/lib/usr/clib/thread \
                           @TOPDIR@/lib/usr/clib/timer \
                           @TOPDIR@/lib/usr/clib/utils \
-                          @TOPDIR@/lib/usr/clib/vec \
```

Comment:
If you rebase this onto main we can merge now.

Reply:
It looks like would have to redo this PR on main instead of the vec code, I assumed that is what you meant. I will recreate this PR based on main.